lib/smart_proxy_remote_execution_ssh/session.rb

```diff
@@ -121,7 +121,7 @@ def local_command_file(filename)
     end
 
     def remote_command_dir
-      File.join(@remote_working_dir, @command.id)
+      "#{@remote_working_dir}-#{@command.id}"
```

Comment:
I wonder, if the `@remote_working_dir` shouldn't be just `/var/tmp`, as there wouldn not be `/var/tmp/foreman-proxy-ssh` directory created, but `/var/tmp/foreman-proxy-ssh-#{uuid}`. So it would be:

```ruby
File.join(@remote_working_dir, "foreman-proxy-ssh-command-#{@command.id}")
```